controller/src/beerocks/master/tasks/dynamic_channel_selection_task.cpp

```diff
@@ -52,41 +52,54 @@ beerocks::eChannelScanErrCode dynamic_channel_selection_task::dcs_request_scan_t
 {
     // When a scan is requested send the scan parameters Channel pool & Dwell time
 
+    auto radio_mac_str = beerocks::net::network_utils::mac_to_string(m_radio_mac);
+
+    //Create request
     auto request = beerocks::message_com::create_vs_message<
         beerocks_message::cACTION_CONTROL_CHANNEL_SCAN_TRIGGER_SCAN_REQUEST>(cmdu_tx);
     if (!request) {
         LOG(ERROR) << "Failed building message cACTION_CONTROL_CHANNEL_SCAN_TRIGGER_SCAN_REQUEST";
         return beerocks::eChannelScanErrCode::CHANNEL_SCAN_INTERNAL_FAILURE;
     }
+    request->scan_params().radio_mac = m_radio_mac;
 
+    //Get dwell time, if dwell time invalid fail scan.
     int32_t dwell_time_msec = database.get_channel_scan_dwell_time_msec(m_radio_mac, m_single_scan);
     if (dwell_time_msec <= 0) {
         LOG(ERROR) << "invalid dwell_time <= 0";
         return beerocks::eChannelScanErrCode::CHANNEL_SCAN_INVALID_PARAMS;
     }
+    request->scan_params().dwell_time_ms = dwell_time_msec;
 
-    //get current channel pool from DB
+    //Get channel pool, if channel pool empty fil scan, if pool is set to "all channel" use
```

Comment:
```suggestion
    // Get channel pool; if channel pool empty: fail scan; if pool is set to "all channel": use
```

Reply:
Fixed typos and updated comment to better describe what happens.